docs/user-docs/latest/content-types-builder/creating-new-content-type.md

```diff
@@ -0,0 +1,50 @@
+# Creating content-types
+
+Strapi allows to create 2 main content-types using the Content-Types Builder: collection types and single types. Although they are not proper content-types as they cannot exist indepentely, components can also be created through the Content-Types Builder, in the same way as collection and single types.
+
+- Collection types are content-types that can be reused several times.
+- Single types are content-types that are created for one purpose and are meant to be used once.
+- Components are a combination of fields handled as a group when configuring the fields of a content-type, and when writing the content of that content-type with the Content Manager.
+
+## Creating a new content-type
+
+![Content-type creation](../assets/content-types-builder/content-type-creation.png)
+
+Content types are created from the Collection types and Single types categories of the Content-Types Builder.
+
+To create a new content-type:
+
+1. Choose whether you want to create a collection type or a single type.
+2. Click on the **Create a new collection/single type** button in the category of the content-type you want to create.
+3. In the content-type creation window, write the name of the new content-type in the *Display name* textbox. When creating a collection type, opt for a singular name, as collection type names are automatically pluralised when displayed in the Content Manager.
```

Comment:
Many thanks for describing this (automatic pluralisation)! 👏 

I found it a bit disturbing when I used Strapi for the first time, for instance in the Quick Start Guide.